definitions/npm/cb-fetch_v1.10.x/test_cb-fetch_v1.10.x.js

```diff
@@ -0,0 +1,30 @@
+import request from 'cb-fetch';
+
+request({ responseType: "json" });
+request().post();
+request().hook('download', () => {});
+request().get().query();
+request().done({
+    success(response) {
+        response.headers.hasOwnProperty();
+    }
+});
+
+// $ExpectError
+request({ responseType: "ms-stream" });
+
+// $ExpectError
+request().trace();
+
+// $ExpectError
+request().hook('upload', () => {});
+
+// $ExpectError
+request().get().send();
+
+request().done({
+    error(response) {
+        // $ExpectError
+        response.headers.hasOwnProperty();
+    }
```

Comment:
Could you please wrap tests in a few [`describe/it` blocks](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope)?

Reply:
How do you display the output using flow-typed? Why bother if it's not used?
I couldn't find a `--verbose` option.